mobile/src/main/java/com/ataulm/wutson/discover/ShowSummaryView.java

```diff
@@ -32,22 +32,23 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
 
     @Override
     protected void onFinishInflate() {
-        View.inflate(getContext(), R.layout.merge_discover_by_genre_item, this);
+        View.inflate(getContext(), R.layout.merge_discover_show_summary, this);
```

Comment:
for custom views that are used in one screen:

- add the prefix (discover) to the name of the view because there are no packages in the resources directory.
- this view is just a ShowSummary view, it doesn't know it's an item in an adapter